### PR TITLE
CONFIG: FET-389 specify NodeJs version for heroku to deploy examples

### PR DESCRIPTION
**Pull request by naibinh**
Guideline:
 - https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: N/A
- gdc-dashboards: N/A

# Related Jira tasks
- FET-389: https://jira.intgdc.com/browse/FET-389

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1224.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5095/ : SUCCESS in 7m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/491/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/492/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2083/ : SUCCESS in 18m 45s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1224.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5096/ : SUCCESS in 7m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/493/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/494/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2084/ : SUCCESS in 17m 25s

**Comment by naibinh** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1224.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5097/ : SUCCESS in 7m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/495/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/496/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2085/ : SUCCESS in 19m 40s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/497/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2086/ : FAILURE in 7m 02s

File: package.json [hunk 1]
```diff
@@ -229,5 +229,8 @@
       "npx gdc-fix-copyright",
       "git add"
     ]
+  },
+  "engines": {
+    "node": "^8.9.4"
```
Comment (ptquang86) [inline package.json]: Please unify the node version
[SDK document](https://sdk.gooddata.com/gooddata-ui/docs/about_gooddataui.html): Node ^8.10.0 or >=9.10.0
[Ticket](https://jira.intgdc.com/browse/FET-389): 8.12.0

Comment (naibinh) [reply package.json]: It's `^8.10.0` now

**Comment by ptquang86** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5098/ : SUCCESS in 7m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/498/ : SUCCESS in 7s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2657/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/501/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2088/ : SUCCESS in 15m 29s

File: package.json [hunk 2]
```diff
@@ -229,5 +229,8 @@
       "npx gdc-fix-copyright",
       "git add"
     ]
+  },
+  "engines": {
+    "node": "^8.10.0"
```
Comment (no23reason) [inline package.json]: Please also update the `.nvmrc` file, because now it points to a version of Node not compatible with this.

Comment (naibinh) [reply package.json]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5100/ : SUCCESS in 4m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/504/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2658/ : SUCCESS in 14m 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5101/ : SUCCESS in 11m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1253/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1688/ : ABORTED in 8m 39s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2659/ : SUCCESS in 11m 51s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5102/ : SUCCESS in 7m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1254/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1690/ : SUCCESS in 18m 54s

**Pull request merged**